packages/ethers/src/client.ts

```diff
@@ -1140,7 +1140,7 @@ export class Web3Modal extends Web3ModalScaffold {
     if (this.authProvider) {
       this.subscribeState(val => {
         if (!val.open) {
-          this.authProvider?.rejectRpcRequest()
+          this.authProvider?.rejectRpcRequests()
```

Comment:
Resolving multiple RPC requests? Oh my

Reply:
Technically only a single one should be open, but it might happen that there are multiple running now (since we don't have the single handler limitation). You can check we keep a list of the open requests and just abort them all.